tests/unit/NameGeneratorTest.php

```diff
@@ -25,7 +25,14 @@ public function testObjectNameForBucketInProjectGenerator(): void
     {
         $nameGenerator = new NameGenerator('KBC_prefix_');
 
-        $this->assertSame('in_c_bucket', $nameGenerator->createObjectNameForBucketInProject('in.c-bucket'));
+        $this->assertSame('in_c_bucket', $nameGenerator->createObjectNameForBucketInProject('in.c-bucket', null));
+    }
+
+    public function testObjectNameForBucketInProjectWithBranchIdGenerator(): void
+    {
+        $nameGenerator = new NameGenerator('KBC_prefix_');
+
+        $this->assertSame('123_in_c_bucket', $nameGenerator->createObjectNameForBucketInProject('in.c-bucket', '123'));
```

Comment:
ja by som sem pre klud duse hodil aj to ked poslem to idcko branche null

Reply:
To je v testu nad tímto